cloud-functions/assets/upload_response.json

```diff
@@ -0,0 +1,78 @@
+{
+    "entry": [
+        {
+            "response": {
+                "etag": "W/\"MTY2Mjc0NTkxNDY4NTAxNTAwMA\"",
+                "lastModified": "2022-09-09T17:51:54.685015+00:00",
+                "location": "https://healthcare.googleapis.com/v1/projects/phdi-357418/locations/us-west1/datasets/phdi-dev-dataset-phdi-357418/fhirStores/phdi-dev-fhirstore/fhir/Patient/3715438f-76f5-4254-ae68-2ab3cfa1e8e6/_history/MTY2Mjc0NTkxNDY4NTAxNTAwMA",
```

Comment:
Is there any concern about having out project_id, dataset, and fhirstore data out in a github repo?  I know this is all test setup and data, but just making sure.  +

Reply:
Good callout. I don't think this is necessarily sensitive data, but I'll go ahead and replace the location with something more generic since it's not used in any tests anyways.